utils.cpp

```diff
@@ -31,7 +31,7 @@ trajectory_t transformTraj(const trajectory_t &traj, const transform_t &tf) {
 bool inRange(const point_t &p, const obstacle_t &obs) {
```

Comment:
We may want to change this comment as well ^^ (to OBSTACLE_MAX_SIZE)

Reply:
I missed that, I'll change that rn